src/plots/cartesian/layout_attributes.js

```diff
@@ -325,7 +325,7 @@ module.exports = {
         ].join(' ')
     },
     tickformat: {
-        valType: 'string',
+        valType: 'any',
```

Comment:
This is the other change added... this was needed, so it lets me pass tickformat as callback